.gitmodules

```diff
@@ -3,4 +3,4 @@
 	url = https://github.com/ethereum/tests.git 
 [submodule "v3-contracts"]
 	path = v3-contracts
-	url = https://github.com/maticnetwork/v3-contracts.git
+	url = https://github.com/maticnetwork/v3-contracts
```

Comment:
Use the new repo https://github.com/0xPolygon/core-contracts?

Reply:
Yep will do. That was the next step. 👍

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/commit/54880bbd64b294f330dcbfbb334a64c1b54a4245